recommendations.md

```diff
@@ -0,0 +1,236 @@
+
+## Release Cadence
+
+Not every project has a stable release cadence, but if you have one (even a rough one), document it.
+It is always better if your release cadence is predictable and aligned with your support lifecycles.
+
+### Good
+
+>There are several release branches for Alpine Linux available at the same time.
+Each May and November we make a release branch from edge.
+The main repository is typically supported for 2 years and the community repository is supported until next stable release.
+
+Source: <https://alpinelinux.org/releases/>
+
+## Explain what's supported
+
+Every project will have differing levels of what counts as "support" - it is important to document what
+"support" means for your project. If there are different tiers (Active/Security/Extended for eg), document
+these clearly.
+
+Bad:
+
+> Extended Support beyond LTS is available to customers on a commercial basis.
+
+Good:
+
+> Extended Support beyond LTS is available to customers on a commercial basis. It includes critical
+security fixes only on the packages within the `base` repository.
+
+> Customers paying for "Premier Support" get additional access to our support team with a guaranteed SLA.
+
+- Good: https://ubuntu.com/security/esm (Clearly explains what ESM means for Ubuntu end users)
+
+## Versioning Policy
+
+Document your versioning policy. Even if the policy is homebrew and varies between major versions, a clearly
+documented policy is better than none.
+
+Good:
+
+> We follow Semantic Versioning, and limit breaking changes to major upgrades.
+
+> The project follows SemVer starting from v12. Prior releases may include breaking changes in minor version upgrades.
+
+## Release Notes
+
+Release notes are critical for your users doing actual upgrades. If certain upgrade pathways are unsupported
+(such as doing 2 major upgrades at once), document it. Highlight breaking changes in your release notes.
+
+If you have a migration guide, link it on all the release notes.
+
+## Listing Releases
+
+List your releases in a table with all the relevant information for each release cycle. This includes:
+
+1. Link to a changelog.
+2. What's the latest release in that cycle.
+3. What are the supported dates for this release (for *all* different support levels)
+4. Download URL, if needed
+5. Release notes
+6. Migration Guide, if available
+
+Prefer listing older/unsupported releases elsewhere (`/historical-releases`). If you think they are important to your
+users, mark them extremely well in the table as unsupported.
+
+## Dates
+
+### Always use absolute dates
+
+Many times, your support/EoL policies are relative. Common examples:
+
+1. The last major release becomes unsupported 90 days after a new major release.
+2. Bug fixes on previous releases will be made till the latest releases gets the first point release.
+
+However, none of this is relevant to your end-users. Make sure that all your releases always have a clear dates
```

Comment:
No hyphen for users = noun (we do hyphenate "end-user documentation", as it's used like an adjective):

```suggestion
However, none of this is relevant to your end users. Make sure that all your releases always have a clear dates
```